### DISCUSSION TITLE
reset(): Calling reset() on an object is deprecated

### DISCUSSION THREAD

**Discussion by ralphjsmit** [livewire/livewire#4534] [7. Bugs (archived)] [answered]
I noticed in my Honeybadger logs that the following error appeared 128 times:

*reset(): Calling reset() on an object is deprecated*

The following backtrace was reported: 

<img width="612" alt="Schermafbeelding 2022-01-17 om 12 06 21" src="https://user-images.githubusercontent.com/59207045/149758729-2a465b19-ff5b-47ea-98b0-217f9986d0e3.png">

I checked out the [PHP website for `reset()`](https://www.php.net/manual/en/function.reset.php) and it turned out that calling `reset()` on an object was deprecated since PHP 8.1.

I'm not sure whether this is a Livewire error or a Laravel error. If this is not a Livewire error, please let me know and I'll open an issue in the Laravel repo.

**Comment by joshhanley** [comment, marked as answer]
@ralphjsmit thanks for reporting! This has also been fixed, in PR #4398, and should be in the 2.9.0 release, can you upgrade to that and see if it fixes the issue for you?

Thanks!

**Reply by ralphjsmit** [reply]
Hey Josh, thank you for your reply! Happy to hear that it's been fixed in 2.9.0. I'll keep an eye on the logs, but I will assume that it's been fixed now. Great!

**Reply by joshhanley** [reply]
@ralphjsmit no worries! 🙂
